sap/rest/gcts/simple.py

```diff
@@ -90,10 +90,13 @@ def pull(connection, name=None, repo=None):
     return repo.pull()
 
 
-def delete(connection, name):
+def delete(connection, name=None, repo=None):
```

Comment:
I cannot see a test for the case where repo is not None.

Reply:
Sample for pull:
- https://github.com/jfilak/sapcli/blob/master/test/unit/test_sap_rest_gcts.py#L736
- https://github.com/jfilak/sapcli/blob/master/test/unit/test_sap_rest_gcts.py#L748